libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -327,6 +327,33 @@ static MultithreadTaskParams_t xGlobalTaskParams[ pkcs11testMULTI_THREAD_TASK_CO
 #define pkcs11testALL_BITS    ( ( 1 << pkcs11testMULTI_THREAD_TASK_COUNT ) - 1 )
 
 
+static CK_RV prvDestroyTestCredentials( void )
```

Comment:
Could you just call `xDestroyCredentials(xGlobalSession)` ?

Reply:
No- I need to distinguish between destroying the test credentials and the regular ones for secure element testing.